### ISSUE TITLE
setting to rename entities missing save button

### ISSUE THREAD

**Issue by npracht** [saelos/saelos#20]
When you go to the settings to rename your entities (for instance `opportunity` to `deal`) you cannot save the page.

**Comment by dbhurley** [comment]
Yep, known issue, this is BETA and those have not been implemented.

**Comment by npracht** [comment]
👍 i can imagine. I just report everything i see.
Is there any type of issue you'd like that we report first (while it is still beta)?

**Comment by npracht** [comment]
Ok clear ! Thanks :)
